core/src/main/kotlin/fr/sncf/osrd/standalone_sim/StandaloneSimulation.kt

```diff
@@ -134,26 +137,30 @@ fun runStandaloneSimulation(
         provisional = provisionalResult,
         finalOutput = finalEnvelopeResult,
         mrsp = makeMRSPResponse(speedLimits),
-        electricalProfiles = makeElectricalProfiles(electrificationMap),
+        electricalProfiles = makeElectricalProfiles(electrificationRanges),
     )
 }
 
 fun makeElectricalProfiles(
-    electrifications: ImmutableRangeMap<Double, Electrification>
+    electrificationRanges: List<ElectrificationRange>
 ): RangeValues<ElectricalProfileValue> {
-    fun profileFromElectrification(electrification: Electrification): ElectricalProfileValue {
+    fun profileFromElectrification(electrification: ElectrificationUsage): ElectricalProfileValue {
         return when (electrification) {
-            is Electrified -> ElectricalProfileValue.Profile(electrification.mode)
+            is ElectrifiedUsage ->
+                ElectricalProfileValue.Profile(
+                    electrification.profile,
+                    electrification.profileHandled
+                )
```

Comment:
We used to have 3 versions: electrified, neutral, not electrified. This merges neutral and not electrified, would it be a problem?